addons/mail/static/src/core/common/messaging_service.js

```diff
@@ -123,6 +123,17 @@ export class Messaging {
         );
     }
 
+    /**
+     * Used by messaging dependant services to start the bus in order to do
+     * so only when `messagingParametersService.start_bus_service` is
+     * set to true. This allow to start the bus only when needed.
+     */
+    startBus() {
+        if (this.env.services["mail.messaging_parameters"].start_bus_service) {
```

Comment:
But I don't like this condition at all. It's useless to register all the handlers if the bus should not even be started.

We should find a better code split to allow registering the handlers and starting the bus only when necessary. (So not in common bundle if livechat needs it but doesn't need the feature).